src/enum.ts

```diff
@@ -1,37 +1,79 @@
+/* Copied from https://github.com/mchehab/zbar, release 0.23.90 */
```

Comment:
I think this is not "Copied" from. How about "Reference zbar.h of release 0.23.90".

Reply:
Well, actually I did copy it from the original source file.

Reply:
Ok... I think this is not a big deal.